Rules Terrain.html

```diff
@@ -0,0 +1,131 @@
+                        <tr>
+                            <td>
+                                <img src="img/terrain/rubble high def.png">
+                                <p>Rubble</p>
+                            </td>
+                            <td> 0 </td> <td> -2 </td> <td> 4 Hexes </td> <td>  </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <img src="img/terrain/rough.png">
+                                <p>Rough Terrain</p>
+                            </td>
+                            <td> None </td> <td> None </td> <td> 4 Hexes </td> <td>  </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <img src="img/terrain/heavy buildings hex.png">
+                                <p>Buildings</p>
+                            </td>
+                            <td> +2 </td> <td> Cannot be shot through </td> <td> Cannot be moved through </td> <td><p> 
+                                Buildings cannot be moved through or shot through. A mech with Jump Jets or the Quantumn
+                                Teleporter can get on top of buildings. While on top of a building, a mech has a +1 to 
+                                accuracy rolls, and all mechs targeting it get a +1. 
+                            </p></td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <img src="img/terrain/lava.png">
+                                <p>Lava</p>
+                            </td>
+                            <td> -1 </td> <td> -1 </td> <td> 1 Hex </td> <td><p> 
+                                For each lava hex entered, a mech takes 2d4+1 damage of the same type as flamethrowers.
+                            </p></td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <img src="img/terrain/Powerplant.png">
+                                <p>Nuclear Power Plant</p>
+                            </td>
+                            <td> 0 </td> <td> 0 </td> <td> 0 </td> <td> <p>
+                                The Nuclear Power Plant has 10 hp. If you destroy the power plant, it explodes as if it has a 
+                                generator level = 2+ (2* round#).
+                            </p> </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <img src="img/terrain/Nuclear Sub.png">
+                                <p>Nuclear Submarine</p>
+                            </td>
+                            <td> 0 </td> <td> 0 </td> <td> 0 </td> <td> <p>
+                                The Nuclear Sub has 10 hp. If you destroy the sub, it explodes as if it has a 
+                                generator level = 3 + (2* round#).
```

Comment:
Would you be upset if we swapped the 3 from the Nuclear Sub with the 2 from the generator? Nuclear power plants are typically significantly larger than what exists in our mechs or on a sub/ship.

Reply:
It's a game balance difference for the water eating 1 of those dice. If you still prefer me pull it back down, then I will.

Reply:
Ah, I see. That's fine